app/views/analytics/districts/show.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 
 <%= render 'shared/cohort_charts', cohort_analytics: @cohort_analytics, quarter: current_quarter, year: current_year %>
-<%= render 'shared/analytics_table',
+<%= render 'analytics/districts/analytics_table',
```

Comment:
Since the District view and the Facility view now show sufficiently different things, we won't share the partials anymore.